staging/src/k8s.io/apimachinery/pkg/runtime/interfaces.go

```diff
@@ -234,9 +234,9 @@ type Object interface {
 // to JSON allowed.
 type Unstructured interface {
 	Object
-	// UnstructuredContent returns a non-nil, mutable map of the contents of this object. Values may be
+	// UnstructuredContent returns a non-nil, map of the contents of this object. Values may be
```

Comment:
Maybe mention that the map is a deep copy of the contents - so that the caller can do whatever it wants to with it.